mandatory/4-tax.js

```diff
@@ -17,7 +21,10 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(firstPrice) {
+  finalPrice = calculateSalesTax(firstPrice);
```

Comment:
This code will work, but we should always use either `let` or `const` when declaring a new variable.